docs/src/pages/customization/breakpoints/breakpoints.md

```diff
@@ -105,7 +105,7 @@ If you are using TypeScript, you would also need to use [module augmentation](/g
 <!-- Tested with packages/material-ui/test/typescript/breakpointsOverrides.augmentation.tsconfig.json -->
 
 ```ts
-declare module '@material-ui/core/styles' {
+declare module '@material-ui/system' {
```

Comment:
Is this a breaking change? How about we try to document API from the core as much as we can? The same way we document the Box from the core, not from the system?

```suggestion
declare module '@material-ui/core/styles' {
```

Or why does it not impact the other theme overrides?

Reply:
Let me re-check.

Reply:
I've reverted these changes, they are not necessary